app/src/main/res/values/strings.xml

```diff
@@ -295,6 +295,8 @@
     <string name="label_system_message_added">%1$s added %2$s to the conversation</string>
     <string name="label_system_message_removed">%1$s removed %2$s from the conversation</string>
     <string name="label_system_message_left_the_conversation">%1$s left the conversation</string>
+    <string name="label_system_message_other_called">%1$s called</string>
+    <string name="label_system_message_you_called">%1$s called</string>
```

Comment:
Do we really need to have 2 different string id's with the same string content?

Reply:
Or maybe we should rename it to?
```suggestion
    <string name="label_system_message_other_called">You received a call from %1$s</string>
    <string name="label_system_message_you_called">%1$s called</string>
```

Reply:
i split it into 2 strings, cause in different languages there could be different texts for "you" and "another person". So in translations it will be used I believe

Reply:
Hmm, but wouldn't the `%1$s` represent the name of the person calling? therefore the translation will be exactly the same

Reply:
this is a message for a missed call, not for a call that is currently running.

Reply:
But when you translate the string, you can't really know in runtime whether the originator of that call is a he/she, so it won't be possible to translate it accordingly anyways.

Reply:
but we know if it's me or somebody else :)

Reply:
lets say: i did it to work in the same was as it's in current app